cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -375,8 +387,12 @@ func (k *Kuberhealthy) monitorExternalChecks(ctx context.Context, notify chan st
 			}
 		}
 
-		// check for changes or additions in the incoming data
-		for _, i := range l.Items {
+		for _, kc := range khChecks.Items {
+			i, err := convertUnstructuredKhCheck(kc)
+			if err != nil {
```

Comment:
add logging here because were deciding to do something based on an error that was returned